### PR TITLE
remove unnecessary files

### PR DESCRIPTION
**Pull request by shibby**

### CODE REVIEW

File: include/constants.php [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<?php
-//
-// This code and all components (c) Copyright 2006 - 2016, Wowza Media Systems, LLC. All rights reserved.
-// This code is licensed pursuant to the Wowza Public License version 1.0, available at www.wowza.com/legal.
-//
-define("BASE_DIR", dirname(dirname(__FILE__)));
```
Comment (shibby) [inline include/constants.php:6]: this constant is only used in `init.php`

File: include/init.php [hunk 2]
```diff
@@ -1,24 +0,0 @@
...
-	$className = array_pop( $arr );
-	$dirs = [
-		'src/com/wowza',
-		'src/com/wowza/entities/application',
-		'src/com/wowza/entities',
-		'src/com/wowza/entities/application/helpers',
-	];
-	foreach ( $dirs as $dir ) {
-		$file = BASE_DIR.'/'.$dir.'/class.'.strtolower( $className ).'.php';
-		if ( file_exists( $file ) ) {
```
Comment (shibby) [inline include/init.php:18]: this file never exists, because dirs above are not exists. 
so we are dealing here with dead code.

**Review by shibby — COMMENTED** [review]
please see inline comments for the reason of removal.

File: tests/users.php [hunk 3]
```diff
@@ -3,7 +3,6 @@
 // This code and all components (c) Copyright 2006 - 2016, Wowza Media Systems, LLC. All rights reserved.
 // This code is licensed pursuant to the Wowza Public License version 1.0, available at www.wowza.com/legal.
 //
-require_once("../include/config.php");
 
 $sf = new com\wowza\User("newuser3");
```
Comment (IWriteThings) [inline tests/users.php:7]: We need to not default to newuser3. We shouldn't be calling a default user like that.

File: tests/streamfile.php [hunk 4]
```diff
@@ -3,7 +3,6 @@
 // This code and all components (c) Copyright 2006 - 2016, Wowza Media Systems, LLC. All rights reserved.
 // This code is licensed pursuant to the Wowza Public License version 1.0, available at www.wowza.com/legal.
 //
-require_once("../include/config.php");
 
 // Create settings
 $settings = new \com\wowza\entities\application\helpers\Settings();
```
Comment (IWriteThings) [inline tests/streamfile.php:8]: The big buck bunny test code below this needs to be placed inside a 'test' function of sorts and make it callable. We shouldn't use a bunch of long lines of commented out code.

**Comment by IWriteThings** [comment]
This is an excellent pull request. I am reviewing it and have found some changes I would like to make or at the least flag for the future. I will try to get this done today if I can. Thank you for the pull request. Good stuff!

**Comment by IWriteThings** [comment]
I haven't forgotten this. Just checking in.

File: tests/streamfile.php (hunk 4 above)
Comment (IWriteThings) [reply tests/streamfile.php:8]: This can be changed later.

**Review by IWriteThings — APPROVED** [review]

**Pull request merged**